e2e/tests/transfer/upgradesv1_test.go

```diff
@@ -0,0 +1,356 @@
+//go:build !test_e2e
```

Comment:
I created a new file for these tests since the original channel creation options were all a `v1` transfer. This was more straightforward than adding conditional logic based on which test was running in the `SetupTest`